src/components/Comments.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import { fetchComments } from "../requests";
+
+export function Comments({ article_id }) {
+  const [commentGroup, setCommentGroup] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    fetchComments(article_id).then(({ data: { comments } }) => {
+      setCommentGroup(comments);
+      setIsLoading(false);
+    });
+  }, []);
+
+  if (isLoading) {
+    return <p>Loading...</p>;
+  }
+  return (
+    <div>
+      <h3>Comments</h3>
+      {commentGroup.map((comment) => {
```

Comment:
This should be in a list element, ul probably for the best and some li with a unique key to get rid of the error you have in console